OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -110,6 +111,8 @@ public class Armament : PausableConditionalTrait<ArmamentInfo>, ITick
 		public readonly WeaponInfo Weapon;
 		public readonly Barrel[] Barrels;
 		Turreted turret;
+		Hovers hovers;
```

Comment:
it feels a bit weird to hardcode `Hovers` here, but I suppose it's fine

Reply:
Wouldn't things like slopes and tilting from explosions also need to modify the offset?